elpis/engines/kaldi/objects/model.py

```diff
@@ -255,3 +256,31 @@ def background_train_task():
         else:
             run_training_in_background()
         return
+
+    def get_train_results(self):
+        log_file = self.path.joinpath('train.log')
+        results = {}
+        with log_file.open() as fin:
+            wer_lines = []
+            for line in reversed(list(fin)):
+                line = line.rstrip()
+                if "%WER" in line:
+                    # use line to sort by best val
+                    line_r = line.replace('%WER ', '')
+                    wer_lines.append(line_r)
+            wer_lines.sort(reverse = True)
+            line = wer_lines[0]
+            line_split = line.split(None, 1)
+            wer = line_split[0]
+            line_results = line_split[1]
+            line_results = line_results.replace('[', '')
+            line_results = line_results.replace(']', '')
```

Comment:
You can use a single `re.sub()` call to replace both of these at once.

Reply:
The [\[\]] in the regex is less readable but as you wish :)